### DISCUSSION TITLE
Hello world

### DISCUSSION THREAD

**Discussion by Oros42** [Oros42/IMSI-catcher#115] [General]
I open a new discussion for everythink that is not an issue :-)

**Comment by Arnoldgenes** [comment]
Echo ./hello_world.sh !! Exactly the platform I was looking for to reach out to you. Man!..I love your repos, projects and site! Im right there with ya and it's awesome to find someone who is leading this crusade. I love this stuff and I'm a big fan...eager to take notes

**Comment by Oros42** [comment]
Thank you :-)
Mmh why, on your github page, you have a link to my personal web site ? 0_o
You didn't have a web site ?

**Reply by Arnoldgenes** [reply]
It's currently in dev as we speak. Honestly I just began navigating your website. None the less I'm psyched to be in contact. I've been studying and researching for my OSCP and am around 2 yes in w serious dev intent. Any direction or pointers you may have I'm more than happy to take on

**Reply by Oros42** [reply]
Don't use wordpress. The source code is horrible.
I recommend you to learn HTML/CSS. And after PHP.
If you want to learn a lot of thing, don't use library, framework or CMS.
Did you pass the CEH ? It's an easy start before OSCP.
